### ISSUE TITLE
Use SauceLabs without Sauce Connect tunnel?

### ISSUE THREAD

**Issue by orcaman** [jmreidy/grunt-mocha-webdriver#88]
The docs say:
####secureCommands
Type: Boolean (Default: false)

If true, it will use saucelabs, with default `hostname` set to `127.0.0.1` and `port` set to `4445`
in order to send selenium commands through Sauce Connect tunnel (more info
[here](https://saucelabs.com/docs/connect#selenium-relay)).

However even when this is set to false, Sauce Connect is used. Is there a way to connect to SauceLabs and run the tests there without Sauce Connect?

**Comment by saadtazi** [comment]
FYI: In PR #81 (not merged yet) I added a `tunneled` (Boolean option).

But I think (not tested) that you can use the following options to run your tests on saucelabs without sauce connect:
```
hostname: 'ondemand.saucelabs.com',
port: 80,
username: 'YOUR_SAUCELAB_USERNAME',
key: 'YOUR_SAUCELAB_KEY'
```

**Comment by orcaman** [comment]
Thanks! will give it a shot and let you know.

**Comment by orcaman** [comment]
Cool, I got this to work using the settings that you provided. A few things that I stumbled along the way:
1. I had to explicitly set "identifier: false" under options in my gruntfile, otherwise I got assigned a tunnel identifier, and my connection was rejected (even though I did not include the secureCommands attribute and did not try to use Sauce Connect)
2. I had to take care that my "testTags" array did not include any undefined values, otherwise my connection would get rejected ("unsupported tags")
